src/Microsoft.AspNetCore.Mvc.Razor/Internal/RazorViewCompiler.cs

```diff
@@ -116,6 +116,8 @@ public class RazorViewCompiler : IViewCompiler
             }
         }
 
+        public bool WatchForChanges { get; set; }
```

Comment:
Suggest this name is too different from `AllowRecompilingViewsOnFileChange`. How about `RecompileViewsOnFileChange`?